mail_tracking/README.rst

```diff
@@ -58,25 +58,25 @@ status icon will appear just right to name of notified partner.
 
 These are all available status icons:
 
-.. |sent| image:: static/src/img/sent.png
+.. |sent| image:: ../static/src/img/sent.png
```

Comment:
Why relative images on main README? This should be on fragments, but not on main README, and the README generator will convert them to absolute. Doing what you have done, will get to work on certain places, but not on others.

Reply:
This regex https://github.com/OCA/maintainer-tools/blob/master/tools/gen_addon_readme.py#L148 doesn't support "image substitutions": https://sublime-and-sphinx-guide.readthedocs.io/en/latest/images.html#use-image-substitutions

Using a regex like this fix the issue:
```
r'\s*\.\. (?:\|.+\|\s)?(figure|image)::\s+(?P<path>.*?)\s*$'
```

Reply:
This is working, @Tardo. What the regexp do is to replace by the absolute URL in raw.github.com, for being accessible no matter where the file is hosted. And that happens when converting fragments to README.rst.

Reply:
You can see it, the actual regex doesn't recognize the image line when use image substitution...

Actual behavior: https://regex101.com/r/Txgcx0/1
With proposed regex: https://regex101.com/r/Txgcx0/2

Parhaps i'm doing something bad... but after change regex i can generate valid readme's.